src/components/Counter.js

```diff
@@ -6,11 +6,17 @@ class Counter extends Component {
     incrementIfOdd = () => {
         // Stretch Problem: Implement an increment function that
         // only increments if the counter value is odd
+        if (this.props.count % 2 !== 0) {
```

Comment:
@CookieMonsta89 good use of the modulo `%` and bang operator `!` to get numbers that aren't even (i.e. odd) 👍